### ISSUE TITLE
Voice Commands enabled by default

### ISSUE THREAD

**Issue by teguria** [IBVSoftware/CollaBoard-Feedback#120]
Surface Hub
v 1.3.16
Time: 11.30 (almost)

I launched CollaBoard on a Hub for a demo to a reseller. The voice started to talk during the "splash screen". The I realized the Voice Commands where enabled, probably because they were already activated on the last session? Anyway the voice starting to talk when you are in the "splash screen" I think it shouldn't happen

**Comment by corradocavalli** [comment]
Voice commands are enabled (if not disabled via settings) at startup since we have to recognize application wide commands like 'Collaboard, settings'
What voice did you hear? Maybe 'can't do this right now?'
@gsantopaolo thoughts on this?

On 21 Feb 2017, at 12:20, alead <notifications@github.com<mailto:notifications@github.com>> wrote:


Surface Hub
v 1.3.16
Time: 11.30 (almost)

I launched CollaBoard on a Hub for a demo to a reseller. The voice started to talk during the "splash screen". The I realized the Voice Commands where enabled, probably because they were already activated on the last session? Anyway the voice starting to talk when you are in the "splash screen" I think it shouldn't happen
